packages/wrapped-keys-lit-actions/tsconfig.lib.json

```diff
@@ -5,7 +5,7 @@
     "declaration": true,
     "types": [],
     "allowJs": true,
-    "checkJs": true
+    "checkJs": false
```

Comment:
Added this because typechecking on the .js files was failing. We should really convert this entire package to TS ASAP.